lib/presentation/localizations/app_localizations.dart

```diff
@@ -446,6 +446,57 @@ class AppLocalizations {
       name: 'Some items could not be copied to Shared space',
```

Comment:
Btw just so you know the name has to be the same as the function name and you can do 
```
flutter pub run intl_translation:extract_to_arb --output-dir=lib/l10n lib/presentation/localizations/app_localizations.dart
```
so it makes the intl_messages.arb

I saw you wrote it by hand that's why! :) 

I fixed these lines in my PR tho

Reply:
oh thanks, you save my hands haha